src/types/ApiRequests.ts

```diff
@@ -19,13 +19,15 @@ export interface AuthRequest {
 }
 
 export interface EditProfileRequest {
-    photoUrl: string | undefined;
+    photoUrl_base64: string | undefined;
     username: string | undefined;
     venmoHandle: string | undefined;
     bio: string | undefined;
 }
 
 export interface CreateUserRequest {
+    username: string;
+    netid: string;
     givenName: string;
     familyName: string;
     photoUrl: string;
```

Comment:
is this alphabetized? idk im too lazy to sing the alphabet in my head rn

Reply:
A
B
C
D
E
FamilyName
GivenName
H
I
J
K
L
M
NetID
O
PhotoUrl
Q
R
S
T
Username
V
W
X
Y
Z

No it's not alphabetized but this is the order that Google sends information back in so I think it's fine to leave it as it is.